src/features/contact/contactverzoek/formulierBeheer/ContactverzoekFormulierBeheerFormulier.vue

```diff
@@ -35,15 +34,36 @@
         v-model="title"
       />
     </label>
+
     <!-- dropdown for afdelingen -->
-    <label class="utrecht-form-label">
+    <label
+      v-if="soort === TypeOrganisatorischeEenheid.Afdeling"
+      class="utrecht-form-label"
+    >
       <span class="required">Afdeling</span>
       <service-data-search
         class="utrecht-textbox utrecht-textbox--html-input"
         :required="true"
         placeholder="Zoek een afdeling"
         :get-data="useAfdelingen"
-        v-model="selectedAfdeling"
+        v-model="selectedOrganisatorischeEenheid"
+        :map-value="(x) => x?.naam"
+        :map-description="(x) => x?.identificatie"
+      />
+    </label>
+
+    <!-- dropdown for groepen -->
+    <label
+      v-else-if="soort === TypeOrganisatorischeEenheid.Groep"
+      class="utrecht-form-label"
+    >
+      <span class="required">Groep</span>
+      <service-data-search
+        class="utrecht-textbox utrecht-textbox--html-input"
+        :required="true"
+        placeholder="Zoek een groep"
+        :get-data="useGroepen"
+        v-model="selectedOrganisatorischeEenheid"
         :map-value="(x) => x?.naam"
         :map-description="(x) => x?.identificatie"
       />
```

Comment:
We hebben componenten voor GroepenSearch en AfdelingenSearch in `\features\contact\contactverzoek\formulier\components`.  Die passen hier volgens mij precies op. Dan zouden we die moeten verplaatsen naar `/features/contact/components`. @mstokericatt kan jij je daar in vinden?

Reply:
in GroepenSearch staat een refactoring suggestie met als doel dat component overbodig te maken. Dus wat mij betreft doen we dat niet en laten we dit nu zo.